client/src/app/recipe-search/recipe-search.component.html

```diff
@@ -17,5 +17,6 @@ <h1>PantryPal</h1>
 
 <app-recipe-list
   [recipeList]="recipeList"
-  [searchedQuery]="searchText"
+  [searchedQuery]="searchedQuery"
+  [isLoading]="isLoading"
```

Comment:
It is handled in search component. Only sent the isLoading over to recipeList to display the 'Loading' text instead of the list of recipes in between searching

Reply:
Yea that loading should appear in the search component, we'll have lots of overlapping code that way.